server/app/controllers/fusor/api/v21/deployments_controller.rb

```diff
@@ -64,12 +64,14 @@ def deploy
       end
     end
 
-    def validate
-      status = @deployment.valid? ? 200 : 422
-
-      render status: status, json: {
-        :errors => @deployment.errors.full_messages,
-        :warnings => @deployment.warnings
+    def validation
```

Comment:
Good point.  Since I was returning a validation object, I wanted the route to be a successful GET on a validation object, so I thought deployments/:id/validation would reflect that.  What do you think about changing the action name back to validate and doing a route like:

```ruby
get :validation, to: '#validate'
```